README.md

```diff
@@ -6,6 +6,20 @@ righteous (**Drake Developers**) who will eventually ride into the human world t
 
 More practically, this repository exemplifies various ways to open a pathway to using [Drake](https://github.com/RobotLocomotion/drake) in your own project.
 
-[![CircleCI](https://img.shields.io/circleci/project/github/RobotLocomotion/drake-shambhala/master.svg)](https://circleci.com/gh/RobotLocomotion/drake-shambhala)
-[![Jenkins](https://img.shields.io/jenkins/s/https/drake-jenkins.csail.mit.edu/job/RobotLocomotion/job/drake-shambhala/job/master.svg)](https://drake-jenkins.csail.mit.edu/job/RobotLocomotion/job/drake-shambhala/)
-[![Travis CI](https://img.shields.io/travis/RobotLocomotion/drake-shambhala/master.svg)](https://travis-ci.org/RobotLocomotion/drake-shambhala)
+# Continuous Integration
+
+Scripts are provided for various CI instances in `scripts/continuous_integration`. The intended purpose of each is described below:
+
+* Circle : exemplifies how to put a project depending on a Drake installation on Circle CI
+* Travis : examplifies how to put a project depending on a Drake installation on Travis via docker image builds
+* Jenkins : provides complete coverage of shambhala for all subprojects and officially supported platforms
+
```

Comment:
Maybe use the directory names:

* `circle_ci`:  ...
* `travis_ci`: ...
* `jenkins`: ...